cookbooks/bcpc-hadoop/templates/default/hb_hbase-site.xml.erb

```diff
@@ -166,4 +166,9 @@
     <value>org.apache.hadoop.hbase.ipc.SecureRpcEngine</value>    
   </property>   
   <% end %>
+
+  <property>
+    <name>hbase.replication</name>
```

Comment:
Do we want to put this property only when `default["bcpc"]["hadoop"]["hbase"]["repl"]["enabled"]` is set to `true`.